src/dbnode/storage/coldflush.go

```diff
@@ -107,7 +107,7 @@ func (m *coldFlushManager) Run(t time.Time) bool {
 		m.Unlock()
 	}()
 
-	m.log.Debug("starting cold flush", zap.Time("time", t))
+	m.log.Debug("starting cold flush", zap.Time("time", t.ToTime()))
```

Comment:
Maybe confirm how often this gets called

Reply:
Seems like this may call decently often, going to put this behind a flag to only log/generate these fields if we're logging at debug level